File: serv/src/main/java/com/example/serv/models/DishDataModel.java
package com.example.serv.models;

import lombok.AllArgsConstructor;

import java.io.Serializable;

@AllArgsConstructor
public class DishDataModel implements Serializable {

    public int idCategory;
    public int idDish;
    public String name;
    public int idImage;
    public int price;
    public int weight;
    public String description;
    public String compose;

}

File: app/src/main/java/com/example/lr2/datamodel/DishDataModel.kt
package com.example.lr2.datamodel


/***
 * Класс с данными для категории блюд
 */
data class DishDataModel (
    var idCategory : Int,  // Категория блюда
    var idDish : Int,  // id (лучше UUID)
    var name : String,  // название
    var idImage : Int,  // id картинки
    var price : Int,  // цена
    var description : String,  // Описание
    var compose : String  // Состав
) : java.io.Serializable